msys2-keyring/PKGBUILD

```diff
@@ -2,23 +2,18 @@
 
 PKGEXT='.pkg.tar.xz'
 pkgname=msys2-keyring
-pkgver=r21.b39fb11
+epoch=1
+pkgver=20201002
```

Comment:
`vercmp.exe 10201002 r21.b39fb11` gives me 1, so an epoch is strictly not needed afaics.